### PR TITLE
Import/Publish :: selected bucket enhancements

### PR DESCRIPTION
**Pull request by medou-boushab**
- Scroll to the selected bucket
- Select the bucket in generic info

### CODE REVIEW

File: app/scripts/proactive/view/CatalogPublishView.js [hunk 1]
```diff
@@ -313,30 +330,36 @@ define(
...
                             localStorage.setItem("selectBucket", that.$('#catalog-publish-buckets-table tr')[0].getAttribute("data-bucketname"));
-                            this.internalSelectBucket(this.$('#catalog-publish-buckets-table tr')[0]);
+                            this.internalSelectBucket(this.$('#catalog-publish-buckets-table tr')[0], true);
                         }
                     }
                 }
             }
         },
+        PublishScrollToBucket: function() {
+            var scrollToVal = $('#catalog-publish-modal .catalog-selected-row').position().top - 300;
```
Comment (tao-xinxiu) [inline app/scripts/proactive/view/CatalogPublishView.js:360]: How is this minus value 300 calculated? Is it always the same in the future?

**Review by tao-xinxiu — APPROVED** [review]

File: app/scripts/proactive/view/CatalogPublishView.js (hunk 1 above)
Comment (medou-boushab) [reply app/scripts/proactive/view/CatalogPublishView.js]: thanks, I use the parent to calculate the scroll position

**Comment by medou-boushab** [comment]
Retest this please

**Review by mboussaa — APPROVED** [review]

**Comment by medou-boushab** [comment]
Retest this please

**Comment by medou-boushab** [comment]
Rest this please

**Pull request merged**